### ISSUE TITLE
Alt and Href not filled in 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [maschek/imgmap#49]
```
What steps will reproduce the problem?
1. Create a map, 
2. Enter 'mycode.asp' in the Href field
3. Enter 'NY' in the Alt Field 
4. Add the co. with mouse, worked Great
5. out put did not ahve the ALT or the HREF I entered
you could see the information in the boxes. but  the output
did not have them. 

What is the expected output? What do you see instead?
<area shape="poly" alt="" title="" 
coords="86,264,94,234,42,154,50,109,5,96,7,168,14,213,51,262" href="" target="" 
/>

What version of the product are you using? On what browser/operating system?


Please provide any additional information below.


```

Original issue reported on code.google.com by `lanefow...@gmail.com` on 18 May 2011 at 10:52
- Blocked on: #46
- Merged into: #46
